src/components/AddComment.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 function AddComment() {
```

Comment:
make here a names export to match the pattern we use in the code
`export function AddComment() {`
and delete default export from the bottom of the file